common/reviews/api/node-core-library.api.md

```diff
@@ -298,8 +299,16 @@ export interface IFileSystemCopyFilesOptions extends IFileSystemCopyFilesAsyncOp
     filter?: FileSystemCopyFilesFilter;
 }
 
+// @public
+export interface IFileSystemCopyFileToManyOptions extends Omit<IFileSystemCopyFileOptions, 'destinationPath'> {
```

Comment:
Can we design this interface without `Omit<>`?  Algebraic types are difficult to represent on an API website.